app.py

```diff
@@ -0,0 +1,89 @@
+from flask import Flask, session, render_template, request, abort, redirect, url_for
+from flask_sqlalchemy import SQLAlchemy
+from functions import get_posts, create_post, get_documents, upload_document, get_forms, upload_form
+import datetime
+import config
+import os
+
+app = Flask(__name__)
+
+app.config.from_object(os.environ['APP_SETTINGS'])
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+db = SQLAlchemy(app)
+
+
+'''from models import Users, Posts, Documents, Forms''' 		# import datatables from postgresql database
+
+
+# API routes for interaction with frontend application
+
+@app.route('/api/posts')							# route description: GET => return all unarchived posts
```

Comment:
If you want to document what a function does, you should use docstrings, as per PEP256. [Here](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings) is an example on how to do it.